### ISSUE TITLE
Location

### ISSUE THREAD

**Issue by QinruShi** [thegoose20/modernistlibraries#15]
The book publish locations in the csv file is hard to clean but this information is important for us. We can try to figure this together and map these location on the map and then compare this information with the places that the bookowners live in to make some conclusion.

**Comment by jiegengsjp** [comment]
Yep! Using different colours to demonstrate information you mentioned is a effective way to compare locations of book publishing and Owners, and it will be easy to find common areas of them. Also, I suggest that we can use light and deep colour to show the length of time or quantity of books.
